atomicapp/nulecule/main.py

```diff
@@ -102,6 +103,40 @@ def __init__(self, app_spec, destination=None,
         self.answers_file = answers_file
         self._process_answers()
 
+    @staticmethod
```

Comment:
A few things:
- Definitely needs comments as to what is going on
- Should we break some of this function up into smaller functions?